### PR TITLE
bug/761_wrong_section_in_orion_mongo_sink

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue https://github.com/telefonicaid/fiware-cygnus/issues/761
* No test are required
* Assignee @frbattid

### CODE REVIEW

File: doc/flume_extensions_catalogue/orion_mongo_sink.md [hunk 1]
```diff
@@ -256,22 +256,20 @@ By default, `OrionMongoSink` has a configured batch size and batch accumulation
 [Top](#top)
 
 ##<a name="section4"></a>Programmers guide
-###<a name="section4.1"></a>`OrionMongoSink` class
-`OrionMongoSink` extends `OrionMongoBaseSink`, which as any other NGSI-like sink extends the base `OrionSink`. The methods that are extended are by `OrionMongoBaseSink` are:
+###<a name="section4.1"></a>`OrionSTHSink` class
+`OrionSTHSink` extends `OrionMongoBaseSink`, which as any other NGSI-like sink, extends the base `OrionSink`. The methods that are extended are:
```
Comment (frbattid) [inline doc/flume_extensions_catalogue/orion_mongo_sink.md]: This is OrionMongoSink, not OrionSTHSink

File: doc/flume_extensions_catalogue/orion_mongo_sink.md [hunk 2]
```diff
@@ -256,22 +256,20 @@ By default, `OrionMongoSink` has a configured batch size and batch accumulation
...
+
+    void persistBatch(Batch batch) throws Exception;
+
+A `Batch` contains a set of `CygnusEvent` objects, which are the result of parsing the notified context data events. Data within the batch is classified by destination, and in the end, a destination specifies the MongoDB collection where the data is going to be persisted. Thus, each destination is iterated in order to compose a per-destination data string to be persisted thanks to any `MongoBackend` implementation.
 
     public void start();
 
-`MongoBackend` is created. This must be done at the `start()` method and not in the constructor since the invoking sequence is `OrionMongoSink()` (contructor), `configure()` and `start()`.
+An implementation of `MongoBackend` is created. This must be done at the `start()` method and not in the constructor since the invoking sequence is `OrionSTHSink()` (contructor), `configure()` and `start()`.
```
Comment (frbattid) [inline doc/flume_extensions_catalogue/orion_mongo_sink.md]: This is OrionMongoSink, not OrionSTHSink

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -5,3 +5,4 @@
 - [BUG] Fix wrong spec date regarding release/0.12.0 (#750)
 - [HARDENING] Add check for null content-type in notifications (#743) 
 - [HARDENING] Add support for configurable number of partitions and replication factor in OrionKafkaSink (#736)
+- [BUG] Fix wrong section about STH class in orion_mongo_sink documentation (#761)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: class --> implementation

File: doc/flume_extensions_catalogue/orion_mongo_sink.md (hunk 1 above)
Comment (pcoello25) [inline doc/flume_extensions_catalogue/orion_mongo_sink.md]: Fixed in a38cf04

File: doc/flume_extensions_catalogue/orion_mongo_sink.md (hunk 2 above)
Comment (pcoello25) [inline doc/flume_extensions_catalogue/orion_mongo_sink.md]: Fixed in a38cf04

File: CHANGES_NEXT_RELEASE (hunk 3 above)
Comment (pcoello25) [inline CHANGES_NEXT_RELEASE]: Fixed in a38cf04

**Comment by frbattid** [comment]
LGTM

**Pull request merged**